debian/changelog

```diff
@@ -0,0 +1,6 @@
+radxa-oobe (0.0.1) unstable; urgency=medium
```

Comment:
```suggestion
radxa-oobe (0.1.0) UNRELEASED; urgency=medium
```
We usually start from 0.1.0. Distro needs to be UNRELEASED or CI will create a release for it.